helpers/azure/api.js

```diff
@@ -910,6 +910,14 @@ var postcalls = {
             properties: ['id'],
             url: 'https://management.azure.com/{id}/configurations?api-version=2017-12-01'
         }
+    },
+    afdSecurityPolicies: {
+        listByProfile: {
+            reliesOnPath: 'profiles.listPostgres',
```

Comment:
```suggestion
            reliesOnPath: 'profiles.listPostgres',
```

Shouldn't this be 'profiles.list'?